frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -0,0 +1,203 @@
+import * as React from 'react';
+import {
+  Checkbox,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateVariant,
+  Select,
+  SelectOption,
+  Title,
+} from '@patternfly/react-core';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { coFetch } from '@console/internal/co-fetch';
+import {
+  getQueryArgument,
+  LoadingBox,
+  LoadingInline,
+  removeQueryArgument,
+  setQueryArgument,
+} from '@console/internal/components/utils';
+import { modelFor, NodeKind, resourceURL } from '@console/internal/module/k8s';
+
+type NodeLogsProps = {
+  obj: NodeKind;
+};
+
+const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
+  const {
+    kind,
+    metadata: { labels, name, namespace: ns },
+    status,
+  } = node;
+  const isWindows = status?.nodeInfo.operatingSystem === 'windows';
+  const pathItems = ['journal'];
+  isWindows
+    ? pathItems.push('containers', 'hybrid-overlay', 'kube-proxy', 'kubelet')
```

Comment:
Note I intentionally omitted `'pods'` as there is another directory level for these that would require a third dropdown, so I figured we'd punt on it for now.